packages/core/src/animation/Animator.ts

```diff
@@ -282,10 +291,20 @@ export class Animator extends Component {
         const { property } = curve;
         const { instanceId } = targetEntity;
 
+        let needRevert = false;
+        const baseAnimatorLayerData = this._animatorLayersData[0];
+        if (!(baseAnimatorLayerData[instanceId] && baseAnimatorLayerData[instanceId][property])) {
```

Comment:
`baseAnimatorLayerData[instanceId]`?

baseAnimatorLayerData is a class instance